nuxt.config.js

```diff
@@ -86,12 +98,21 @@ module.exports = {
   plugins: [
     '@/plugins/i18n',
     '@/plugins/element-ui',
+    '@/plugins/axios',
     {src: '@/plugins/clipboard', ssr: false},
-    {src: '@/plugins/error-handler', ssr: false}
+    {src: '@/plugins/error-handler', ssr: false},
+    {src: '@/plugins/persistedstate', ssr: false}
   ],
   modules: [
-    '@nuxtjs/webpackmonitor',
-    '@nuxtjs/axios'
+    '@nuxtjs/axios',
+    ['@nuxtjs/dotenv', { only: [
+      'HOST',
+      'ENDPOINT_BACKEND_AUTH',
+      'ENDPOINT_BACKEND_VALIDATE',
+      // See also server/utils/environment-variables.js
+      'LB_ADDR',
+      'SHOW_EXAMPLES'
```

Comment:
I'm still unsure how to use this in Nuxt side. (incomplete)
An alternate would be to use for Nuxt and environment variable injection, either this and/or with an environment variable exposition Side-Car endpoint (see `/hpi/env`).
A sure thing would be that private data would NOT BE a good fit for unauthenticated `/hpi/env`, and for Nuxt side, this module here would be a good fit.